apps/etterlatte-behandling/src/test/kotlin/BehandlingIntegrationTest.kt

```diff
@@ -328,100 +329,50 @@ abstract class BehandlingIntegrationTest {
     }
 
     protected val tokenSaksbehandler: String by lazy {
-        issueToken(
-            mapOf(
-                "navn" to "John Doe",
-                Claims.NAVident.toString() to "Saksbehandler01",
-                "groups" to listOf(azureAdSaksbehandlerClaim),
-            ),
-        )
+        server.issueSaksbehandlerToken(navn = "John Doe", navIdent = "Saksbehandler01", groups = listOf(azureAdSaksbehandlerClaim))
     }
 
     protected val tokenSaksbehandler2: String by lazy {
-        issueToken(
-            mapOf(
-                "navn" to "Jane Doe",
-                Claims.NAVident.toString() to "Saksbehandler02",
-                "groups" to listOf(azureAdSaksbehandlerClaim),
-            ),
-        )
+        server.issueSaksbehandlerToken(navn = "Jane Doe", navIdent = "Saksbehandler02", groups = listOf(azureAdSaksbehandlerClaim))
     }
 
-    protected val fagsystemTokenEY: String by lazy {
-        issueToken(
-            mapOf(
-                "navn" to "fagsystem",
-                Claims.NAVident.toString() to Fagsaksystem.EY.navn,
-                "groups" to listOf(azureAdSaksbehandlerClaim, azureAdAttestantClaim),
-            ),
-        )
-    }
+    protected val fagsystemTokenEY: String by lazy { server.issueSystembrukerToken() }
```

Comment:
Kan hende det her er riktig - men er vel ikke det samme claims?

Den gamle laget en saksbehandlertoken med navn EY - den nye lager en ekte systembruker token

Reply:
Ja, sant, det er endra. Det er med vilje, sia EY er systembrukar, og det å ha det som ein saksbehandlar var (iallfall for meg) berre forvirrande.